techniques/system/server-roles/1.0/password-check.st

```diff
@@ -271,8 +271,7 @@ bundle agent root_password_check_dav
 
   classes:
 
-      "dav_cant_connect" not => returnszero("/usr/bin/curl -s -f -u ${g.davuser}:${g.davpw} -T /opt/rudder/etc/uuid.hive http://localhost/inventory-updates/uuid.hive","noshell");
-
+      "dav_cant_connect" not => returnszero("/usr/bin/curl -s -f -o /dev/null -u ${g.davuser}:${g.davpw} -T /opt/rudder/etc/uuid.hive http://localhost/inventory-updates/uuid.hive","noshell");
```

Comment:
We need to send the output to /dev/null here because CFEngine sends stdout from returnszero commands to it's own stdout, making the output messy